code/ui/blocks/src/blocks/Title.tsx

```diff
@@ -3,21 +3,72 @@ import type { FunctionComponent, ReactNode } from 'react';
 import React, { useContext } from 'react';
 import { Title as PureTitle } from '../components';
 import { DocsContext } from './DocsContext';
+import type { Of } from './useOf';
+import { useOf } from './useOf';
 
 interface TitleProps {
+  /**
+   * Specify where to get the title from. Must be a CSF file's default export.
+   * If not specified, the title will be read from children, or extracted from the meta of the attached CSF file.
+   */
+  of?: Of;
+
+  /**
+   * Specify content to display as the title.
+   */
   children?: ReactNode;
 }
 
 const STORY_KIND_PATH_SEPARATOR = /\s*\/\s*/;
 
-export const extractTitle = (title: ComponentTitle) => {
+export const getTitleFromContext = (title: ComponentTitle) => {
```

Comment:
Can you check if this is exported outside of the package?
If it is indeed exported, then changing it's name is technically a breaking change.

Reply:
Correct, it's used elsewhere, I didn't spot it was exported.

* [ ] Revert to previous name

Reply:
Reverted!